gulpfile.js

```diff
@@ -56,15 +65,24 @@ gulp.task('copy-static-assets-from-design-system', () => {
     .pipe(gulp.dest(`${OUTPUT_DIRECTORY}/favicons`))
 })
 
+gulp.task('build-styles', () => {
+  return gulp
+    .src('./src/scss/*.scss')
+    .pipe(sassCompiler(sassOptions).on('error', sassCompiler.logError))
+    .pipe(gulp.dest('./dist/assets/css'))
+});
+
 gulp.task('build-script', gulp.series(...scripts.map(createBuildScriptTask)))
 
 gulp.task('watch-and-build', async () => {
     gulp.watch('./src/js/**', gulp.series('build-script'))
+    gulp.watch('./src/scss/*.scss', gulp.series('build-styles'))
 })
 
+
 gulp.task(
   'build',
-  gulp.series('build-script', 'copy-static-assets-from-design-system')
+  gulp.series('build-script', 'build-styles', 'copy-static-assets-from-design-system')
 )
 
-gulp.task('watch', gulp.series('build-script', 'copy-static-assets-from-design-system', 'watch-and-build'))
+gulp.task('watch', gulp.series('build-script', 'build-styles', 'copy-static-assets-from-design-system', 'watch-and-build'))
```

Comment:
If this task is just running the `build` task followed by `watch-and-build` maybe we should just have

```
gulp.task('watch', gulp.series('build, 'watch-and-build'))
```

This way if the build task changes we don't need to worry about updating the watch task